include/grpcpp/security/credentials_impl.h

```diff
@@ -250,6 +253,10 @@ class MetadataCredentialsPlugin {
       grpc::string_ref service_url, grpc::string_ref method_name,
       const grpc::AuthContext& channel_auth_context,
       std::multimap<grpc::string, grpc::string>* metadata) = 0;
+
+  virtual grpc::string DebugString() const {
```

Comment:
This is a `const` method and the one on `CallCredentials` is not `const`. Should we make them consistent?